scrapy/crawler.py

```diff
@@ -123,22 +127,21 @@ def __init__(self, settings):
         super(CrawlerProcess, self).__init__(settings)
         install_shutdown_handlers(self._signal_shutdown)
         self.stopping = False
-        self.log_observer = log.start_from_settings(self.settings)
-        log.scrapy_info(settings)
+        configure_logging(settings)
```

Comment:
I agree, but CrawlerProcess is intended to be a ScrapyCommand util, which makes Scrapy behave as a framework (it starts the reactor too). On #873 we're going to move this class to another place and this distinction would be hopefully more obvious. I plan to document that using `configure_logging` without settings doesn't create any handlers, for custom script purposes.